examples/textures/textures_image_channel.c

```diff
@@ -0,0 +1,99 @@
+/*******************************************************************************************
+*
+*   raylib [textures] example - Retrive image channel (mask)
+*
+*   NOTE: Images are loaded in CPU memory (RAM); textures are loaded in GPU memory (VRAM)
+*
+*   Example originally created with raylib 5.1-dev, last time updated with raylib 5.1-dev
+*
+*   Example contributed by Bruno Cabral (github.com/brccabral) and reviewed by Ramon Santamaria (@raysan5)
+*
+*   Example licensed under an unmodified zlib/libpng license, which is an OSI-certified,
+*   BSD-like license that allows static linking with closed source software
+*
+*   Copyright (c) 2024-2024 Bruno Cabral (github.com/brccabral) and Ramon Santamaria (@raysan5)
+*
+********************************************************************************************/
+
+#include <raylib.h>
+
+//------------------------------------------------------------------------------------
+// Program main entry point
+//------------------------------------------------------------------------------------
+int main(void)
+{
+    // Initialization
+    //--------------------------------------------------------------------------------------
+
+    const int screenWidth = 800;
+    const int screenHeight = 612;
+
+    InitWindow(screenWidth, screenHeight, "raylib [textures] example - extract channel from image");
+
+    Image fudesumi_image = LoadImage("resources/fudesumi.png");
```

Comment:
Please, could you review [raylib naming conventions](https://github.com/raysan5/raylib/blob/master/CONVENTIONS.md), raylib does not use `snake_case` style on variables.